packages/mobile/src/components/lineup-tile/CollectionTileTrackList.tsx

```diff
@@ -123,7 +129,7 @@ export const CollectionTileTrackList = ({
       {tracks.slice(0, DISPLAY_TRACK_COUNT).map((track, index) => (
         <TrackItem
           key={track.uid}
-          active={playingUid === track.uid}
+          uid={track.uid}
```

Comment:
so the main idea here is that we don't rerender every tile when playinguid changes yeah?

Reply:
Yes, this line is specific to the items in the collection tile but the other changes in the pr achieve what you said